File: Java/PC.java
public class PC {
    private Processor processorObj;  // menampung processor
    private RAM ramObj;     // menampung RAM
    private Disk diskObj;   // menampung disk
    private int totalPrice; // menampung total harga

    // constructor
    public PC() {
    }

    public PC(Processor processorObj, RAM ramObj, Disk diskObj, int totalPrice) {
        this.processorObj = processorObj;
        this.ramObj = ramObj;
        this.diskObj = diskObj;
        this.totalPrice = totalPrice;
    }

    // setter and getter
    public void setProcessor(Processor processorObj)
    {
        this.processorObj = processorObj;
    }

    public Processor getProcessor()
    {
        return this.processorObj;
    }

    public void setDisk(Disk diskObj)
    {
        this.diskObj = diskObj;
    }

    public Disk getDisk()
    {
        return this.diskObj;
    }

    public void setRAM(RAM ramObj)
    {
        this.ramObj = ramObj;
    }

    public RAM getRAM()
    {
        return this.ramObj;
    }

    public void setTotalPrice(int totalPrice)
    {
        this.totalPrice = totalPrice;
    }

    public int getTotalPrice()
    {
        return this.totalPrice;
    }

    // method
    public void printPC() {
        System.out.println("Computer Specification :");
        this.processorObj.printProcessor();
        this.ramObj.printRAM();
        this.diskObj.printDisk();
        System.out.println("Total Price         : Rp " + this.getTotalPrice() + ",-");
    }
}

File: Python/PC.py
from functools import total_ordering
from platform import processor
from Processor import Processor
from RAM import RAM
from Disk import Disk

class PC:
    __processorObj = Processor()    # menampung processor
    __ramObj = RAM()                # menampung ram
    __diskObj = Disk()              # menampung disk
    __totalPrice = 0                # menampung harga total

    # constructor
    def __init__(self, ProcessorObj, ramObj, diskObj, totalPrice = 0):
        self.__processorObj = ProcessorObj
        self.__ramObj = ramObj
        self.__diskObj = diskObj
        self.__totalPrice = totalPrice

    # setter and getter
    def setProcessor(self, processorObj):
        self.__processorObj = processorObj
    
    def getProcessor(self):
        return self.__processorObj

    def setRAM(self, ramObj):
        self.__ramObj = ramObj

    def getRAM(self):
        return self.__ramObj

    def setDisk(self, diskObj):
        self.__diskObj = diskObj
    
    def getDisk(self):
        return self.__diskObj
    
    def setTotalPrice(self, totalPrice):
        self.__totalPrice = totalPrice
    
    def getTotalPrice(self):
        return self.__totalPrice

    # method
    def printPC(self):
        print("Computer Specification:")
        self.__processorObj.printProcessor()
        self.__ramObj.printRAM()
        self.__diskObj.printDisk()
        print("Total Price      : Rp " + str(self.getTotalPrice()) + ",-")